app/views/shipit/merge_status/_merge_queue_button.html.erb

```diff
@@ -1,16 +1,28 @@
-<% classes = 'branch-action-btn float-right js-immediate-updates js-handle-pull-merging-errors' %>
+<% classes = 'js-immediate-updates js-handle-pull-merging-errors' %>
 <% if pull_request.try!(&:waiting?) %>
   <%= form_tag dequeue_pull_request_path(stack, pull_request_number), method: 'delete', class: classes, data: {remote: true} do %>
     <%= hidden_field_tag 'referrer', params[:referrer] %>
-    <button type="submit" data-disable-with="Removing from merge queue…" class="btn">
-      Remove from merge queue
-    </button>
+    <span class='branch-action-btn float-right'>
+      <button type="submit" data-disable-with="Removing from merge queue…" class="btn">
+        Remove from merge queue
+      </button>
+    </span>
   <% end %>
 <% else %>
-  <%= form_tag enqueue_pull_request_path(stack, pull_request_number), method: 'put', class: classes, data: {remote: true} do %>
+  <%= form_tag enqueue_pull_request_path(stack, pull_request_number), method: 'put', class: classes, data: {remote: true}, id: 'merge-form' do %>
     <%= hidden_field_tag 'referrer', params[:referrer] %>
-    <button type="submit" data-disable-with="Adding to merge queue…" class="btn btn-primary">
-      Add to merge queue
-    </button>
+
+    <div class='rollbackability'>
+      <%= label_tag :unsafe_to_rollback do %>
+        This change is <%= link_to('unsafe to rollback', 'https://development.shopify.io/guides/shipping/safe_to_merge#Signs_your_PR_is_safe_to_rollback', target: :_blank) %>
```

Comment:
Todo: Find a public source or make this link configurable, as the current link is only available to shopify employees